File: app/src/main/java/com/concough/android/models/EntranceOpenedCountModel.java
package com.concough.android.models;

import io.realm.RealmObject;

/**
 * Created by abolfazl on 7/16/17.
 */

public class EntranceOpenedCountModel extends RealmObject {
    public String entranceUniqueId = "";
    public int count = 1;
    public String type = "";
}

File: app/src/main/java/com/concough/android/models/EntranceOpenedCountModel.kt
package com.concough.android.models

import io.realm.RealmObject

/**
 * Created by abolfazl on 7/12/17.
 */
class EntranceOpenedCountModel: RealmObject() {
    var entranceUniqueId: String = ""
    var count: Int = 1
    var type: String = ""
}